packages/edit-post/src/index.js

```diff
@@ -137,6 +121,26 @@ export function initializeEditor(
 		} );
 	}
 
+	/*
+		Prevent adding template part in the post editor.
+		Only add the filter when the post editor is initialized, not imported.
+		Also only add the filter(s) after registerCoreBlocks()
+		so that common filters in the block library are not overwritten.
+	 */
```

Comment:
Let's follow JS Doc standard for the multiline comments - https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#multi-line-comments

Reply:
Good pick up, thank you!